src/cpp/session/modules/SessionRMarkdown.R

```diff
@@ -357,8 +357,16 @@
       })
    }
    
+   # define handler for logical values
+   handlers <- list(
+      logical = function(x) {
+         value <- ifelse(x %in% TRUE, "true", "false")
```

Comment:
`x %in% TRUE` also helps us handle `NA` values, which we just emit as `false`.

Reply:
nice!